src/sass/Countries.scss

```diff
@@ -0,0 +1,119 @@
+$primary-color: #5588e3;
+$secondary-color: #ffff;
+$cardA-color: #4368af;
+$cardB-color: #3e63a5;
+$breakpoint-phonelg: 470px;
+$breakpoint-tablet: 768px;
+
+.world-card {
+  display: flex;
+  flex-direction: row;
+  background-color: $primary-color;
+  margin: 0 auto;
+  border-radius: 0;
+  height: 11rem;
+  color: $secondary-color;
+
+  .world-img {
+    width: 60%;
+  }
+
+  .world-body {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .world-body p {
+    font-size: 15px;
+  }
+}
+
+.samble-line {
+  color: $secondary-color;
+  padding: 4px 10px;
+  margin: 0;
+  width: 100%;
+  background-color: #35548b;
+}
+
+.card-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  width: 100%;
+  gap: 6px;
+  padding: 6px 5px;
+  text-align: right;
+}
+
+.card-container a {
+  color: white;
+  text-decoration: none;
+  padding: 8px 10px;
+  width: 100%;
+}
+
+.card_Country {
+  position: relative;
+  // padding: 0.625rem;
+}
```

Comment:
- [ ] [OPTIONAL] Great job with your project 👏.  Kindly remove this commented line of code 🙏

Reply:
Thanks for the reminder @Ugboaja-Uchechi. 😄